examples/dgl/ogbn_products_sage_quiver.py

```diff
@@ -73,17 +69,18 @@ def inference(self, g, x, device):
             sampler = dgl.dataloading.MultiLayerFullNeighborSampler(1)
             dataloader = dgl.dataloading.NodeDataLoader(
                 g,
-                th.arange(g.num_nodes()),
+                th.arange(g.num_nodes(), device=g.device),
                 sampler,
-                batch_size=args.batch_size,
-                shuffle=True,
+                device=device,
+                batch_size=args.batch_size*4,
```

Comment:
Why batch size * 4 here?

Reply:
Use a larger batch size for evaluation, following the PyG version that uses a batch size of 1024 for training while 4096 for validation.